spec/features/plugins_spec.rb

```diff
@@ -0,0 +1,91 @@
+# frozen_string_literal: true
+
+describe "Plugins" do
+  # rubocop:disable RSpec/LeakyConstantDeclaration
+  specify "plguin regsitration, load and resolving" do
```

Comment:
That's... a really big specify. Maybe separate it in smaller chunks? And also create stuff like ` InternalTestPluginInterceptor` outside of it?

Reply:
well, these are well-known table_sync specs :)
but - ok :) i will split them